CHANGES

```diff
@@ -17,6 +17,10 @@ Features added
 * ``VerbatimHighlightColor`` is a new
   :ref:`LaTeX 'sphinxsetup' <latexsphinxsetup>` key (refs: #4285)
 * Easier customizability of LaTeX macros involved in rendering of code-blocks
+* Add :confval:`smart_quotes` to disable smart quotes through ``conf.py``
```

Comment:
I noticed `smartquotes_disables` does not mentioned here.

Anyway, these settings for smart quotes are named `smart_quotes` or `smartquotes`. I feel they should have same prefix. What do you think?

Reply:
`smartquotes_excludes` is mentioned at next line ...

I agree about using same prefix `smartquotes` for all. Docutils has a `smart_quotes` or `smart-quotes` option and this why I originally used `smart_quotes`. But Docutils has the `smartquotes_action` class attribute hence I also followed. But I feel `smartquotes` prefix for all conf.py variables is much better indeed.

Reply:
oh, sorry > `smartquotes_excludes`